DataFormats/METReco/src/classes_def.xml

```diff
@@ -159,12 +159,16 @@
   <class name="std::vector<reco::PhiWedge>"/>
   <class name="std::vector<reco::PhiWedge>::iterator"/>
 
-  <class name="reco::EcalHaloData" ClassVersion="10">
+  <class name="reco::EcalHaloData" ClassVersion="12">
+   <version ClassVersion="12" checksum="2557428940"/>
+   <version ClassVersion="11" checksum="2423056450"/>
```

Comment:
intermediate versions from private development steps should be removed.
If the latest version has checksum 2557428940, give it ClassVersion 11 and drop the intermediate ClassVersion 11